src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -1693,6 +1709,94 @@ else if (scale instanceof Time) {
     ZarrGroup subGroup = ZarrGroup.create(subGroupPath);
     Map<String, Object> attributes = new HashMap<String, Object>();
     attributes.put("multiscales", multiscales);
+
+    if (omeroMetadata) {
+      int seriesIndex = seriesList.indexOf(series);
+
+      Map<String, Object> omero = new HashMap<String, Object>();
+      String name = meta.getImageName(seriesIndex);
+      if (name == null) {
+        name = "Series " + seriesIndex;
+      }
+      omero.put("name", name);
```

Comment:
As this is loosely defined, I would be rather inclined to store this metadata value in the `multiscales.name` attributes - see https://ngff.openmicroscopy.org/0.4/#multiscale-md

Reply:
Fixed by 1729af5